app/components/Auth/Form.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import {
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+} from 'react-native';
+import Button from 'react-native-button';
+
+class Form extends React.Component {
+  constructor() {
+    super();
```

Comment:
this needs `super(props)`

Reply:
If I do a property initializer won't I have to remove this line?